src/OrbitQt/TargetLabelTest.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <memory>
+
+#include "OrbitGgp/Instance.h"
+#include "TargetLabel.h"
+#include "process.pb.h"
+
+namespace orbit_qt {
+
+namespace fs = std::filesystem;
+
+TEST(TargetLabel, Constructor) {
+  TargetLabel label{};
+  EXPECT_TRUE(label.text().isEmpty());
+  EXPECT_TRUE(label.styleSheet().isEmpty());
```

Comment:
I'm not saying you should remove it, only to think about it if this couldn't be checked differently.

It looks a lot like an implementation detail to me. In the end for the user of the class it should be of no concern
how the font coloring is done. You could also use QPalette, or overwrite the paintEvent method, or even role your own label implementation. The only important thing in the end is the color of the text, not the stylesheet.

And by putting that check into the tests you even guarantee to the user that this is officially observable behaviour. That means someone could start checking the stylesheet property to figure out in which mode the target label currently is. Still unlikely for this use-case, but I hope you get the idea.

Reply:
As we talked offline, I changed the testing of this. It now uses QPalette instead of spreadsheet and I only test for change in color.